src/algebraic_geometry/prime_spectrum/maximal.lean

```diff
@@ -73,4 +74,60 @@ instance : t1_space $ maximal_spectrum R :=
 
 lemma to_prime_spectrum_continuous : continuous $ @to_prime_spectrum R _ := continuous_induced_dom
 
+variables (R) [is_domain R] {K : Type v} [field K] [algebra R K] [is_fraction_ring R K]
+
+/-- An integral domain is equal to the intersection of its localizations at all its maximal ideals
+viewed as subalgebras of its field of fractions. -/
+theorem localization_infi_eq_bot :
```

Comment:
I think it should be called `infi_localization_eq_bot` instead according to the naming convention.